File: CS50x/pset6/cash/cash.py
from cs50 import get_float

change = get_float("Change owed: ")
while change < 0:
    change = get_float("Change owed: ")

change = round(change * 100)
count = 0

while change != 0:
    if change >= 25:
        change = change - 25
        count += 1
        continue
    elif change >= 10:
        change = change - 10
        count += 1
        continue
    elif change >= 5:
        change = change - 5
        count += 1
        continue
    change = change - 1
    count += 1
print(count)

File: CS50x/pset1/cash/cash.c
#include <stdio.h>
#include <cs50.h>
#include <math.h>

int main(void)
{
    float change;
    do
    {
        change = get_float("Change Owed: ");
    }
    while (change < 0);
    int cents = round(change * 100);
    int coins = 0;
    while (cents != 0)
    {
        if (cents >= 25)
        {
            cents = cents - 25;
            coins++;
            continue;
        }
        if (cents >= 10)
        {
            cents = cents - 10;
            coins++;
            continue;
        }
        if (cents >= 5)
        {
            cents = cents - 5;
            coins++;
            continue;
        }
        if (cents >= 1)
        {
            cents = cents - 1;
            coins++;
            continue;
        }
    }
    printf("%i\n", coins);
}